test/e2e/scenarios/ACL/acl_create_update.yaml

```diff
@@ -102,6 +102,8 @@ steps:
             status: "True"
             timeout: 180
     expect_k8s:
+      status:
+        events:
```

Comment:
I don't see any events in the scenarios. Is it intentional?

Reply:
Because `events` are generated by service according to behaviors of a resource. We cannot predict the content of `events`. Hence, the custom logic adding to test framework only validate if there is something inside of `events` instead of validating what exactly in `events`.

Reply:
Also, because of the way `runtime` works, k8s resource always has one less event than aws resource.

Reply:
If we add `requeue_on_success_seconds`, then we can continue to populate events as they become available. See how SageMaker uses it - https://github.com/aws-controllers-k8s/sagemaker-controller/blob/main/generator.yaml#L64-L65

The docs have a section about this, so you can understand what it does - https://aws-controllers-k8s.github.io/community/docs/contributor-docs/code-generator-config/#reconcile-controlling-reconciliation-and-requeue-logic

Reply:
Added.

Reply:
Removed `requeue_on_success_seconds`. It caused a lot of panic and made a lot of tests failed.

Reply:
Oh no. Okay we can revisit this some other time